go/install/install_darwin.go

```diff
@@ -286,10 +290,12 @@ func installKeybaseService(context Context, service launchd.Service, plist launc
 }
 
 // UninstallKeybaseServices removes the keybase service (includes homebrew)
-func UninstallKeybaseServices(runMode libkb.RunMode, log Log) error {
+func UninstallKeybaseServices(context Context, log Log) error {
+	runMode := context.GetRunMode()
+	err0 := fallbackKillProcess(context, log, defaultServiceLabel(runMode, false), context.GetServiceInfoPath(), "")
```

Comment:
so we always try the fallback kill?  won't that create a lot of spurious warnings?

Reply:
Just one log message that no fallback pid file exists.